lighthouse-core/gather/gatherers/dobetterweb/all-event-listeners.js

```diff
@@ -118,18 +118,16 @@ class EventListeners extends Gatherer {
     })).then(nestedListeners => [].concat(...nestedListeners));
   }
 
-  beforePass(options) {
```

Comment:
>  basically, is there a legitimate reason to still use them?

no. you can use mutation observers. but the events are from a dark time.
only really shitty legacy code should be using them. i hope.